lib/chef/resource/powershell_package.rb

```diff
@@ -44,6 +44,10 @@ class PowershellPackage < Chef::Resource::Package
         description: "Skip validating module author.",
         default: false, introduced: "14.3", desired_state: false
 
+      property :allow_clobber,  [TrueClass, FalseClass],
+        description: "Overrides warning messages about installation conflicts about existing commands on a computer.",
+        default: false
```

Comment:
@mikef-nl Can you also add `introduced: "18.5"` so the `property` similar to the above properties?

Reply:
Have added it as requested